### DISCUSSION TITLE
Difficulty in config file

### DISCUSSION THREAD

**Discussion by Sohandas123** [open-quantum-safe/oqs-demos#221] [Q&A]
Actually I don't want a docker image for oqs-openvpn, and that's why I followed commands written in Docker file to integrate oqs-provider with openssl. I have generated all the required certs and key using commands written in createcerts_and_config.sh. Then I run server.config in one terminal and after that I run client.config from another terminal of the same pc. I noticed some faults like tls handshake failed, key negotiation failed. Can anyone suggest me what things should I change in those config files ?

**Comment by baentsch** [comment]
> Can anyone suggest me what things should I change in those config files ?

Without knowing the error messages or knowing the integration path you took this is hard-to-impossible.

> Actually I don't want a docker image for oqs-openvpn

Sorry to hear that -- they've been created exactly to avoid such build-and-config errors that are pretty much impossible to debug remotely.
